spec/models/spree/line_item_spec.rb

```diff
@@ -623,22 +617,28 @@ module Spree
           end
 
           context "and final_weight_volume has been changed" do
-            before { li.update_attribute(:final_weight_volume, 35) }
+            before do
+              li.final_weight_volume = 35
+            end
```

Comment:
why do we need the multiline though? it's actually simpler than what we had. It's just a matter of preference though.

Reply:
Just changed it back to a single-line block.